src/datepicker/__tests__/datepickers-composed-range.scenario.js

```diff
@@ -29,7 +29,7 @@ function printTime(dt) {
 
 export function Scenario() {
   const [css, theme] = useStyletron();
-  const [dates, setDates] = useState<Array<Date>>([START_DATE, END_DATE]);
+  const [dates, setDates] = useState<Array<?Date>>([START_DATE, END_DATE]);
```

Comment:
This and the changes below may indicate a substantial change required to application code. Do you have an idea of how many instances there are in internal code that would need updating? Try running the alpha test in CI